ceph-releases/jewel/ubuntu/14.04/daemon/ceph.defaults

```diff
@@ -21,9 +21,9 @@
 /mon/mon_force_quorum_join true
 
 #crush
-/global/osd_pool_default_pg_num 32
-/global/osd_pool_default_pgp_num 32
-/global/osd_pool_default_size 3
+/global/osd_pool_default_pg_num 16
```

Comment:
Please divide this commit into two separated commits next time.
Because they are different purpose.